### PR TITLE
Add Michal Laskowski to ndelius-test-automation

### PR DESCRIPTION
**Pull request by marcus-bcl**

### CODE REVIEW

File: terraform/ndelius-test-automation.tf
Comment (AntonyBishop) [inline terraform/ndelius-test-automation.tf:1]: Please can that back to `ndelius-test-automation`. Module needs to be the same as the name of the repo. This will break existing access.

File: terraform/ndelius-test-automation.tf
Comment (AntonyBishop) [inline terraform/ndelius-test-automation.tf:17]: This needs to be changed. Should be in a block as follows:

Comment (AntonyBishop) [reply terraform/ndelius-test-automation.tf:17]: {
      github_user  = "<github username>"
      permission   = "push"  #  pull|push|admin
      name         = ""  #  The name of the person behind github_user
      email        = ""  #  Their email address
      org          = ""  #  The organisation/entity they belong to
      reason       = ""  #  Why is this person being granted access?
      added_by     = ""  #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
      review_after = ""  #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
    },

**Review by AntonyBishop — CHANGES_REQUESTED** [review]
Please see comments.

**Comment by AntonyBishop** [comment]
Hi @marcus-bcl . Grateful if you could replace the current `added_by` value with an MoJ Team contact.

File: terraform/ndelius-test-automation.tf
Comment (AntonyBishop) [inline terraform/ndelius-test-automation.tf:22]: New process changes requires this to me an MoJ contact.

Comment (marcus-bcl) [reply terraform/ndelius-test-automation.tf:22]: No problem, thanks Antony.  Does the contact need to be a GitHub member, or could they approve on Slack or elsewhere if not?

Comment (NickWalt01) [reply terraform/ndelius-test-automation.tf:22]: Hi, it needs to be in the file as a script parses line 22 looking for an MoJ approved email

Comment (marcus-bcl) [reply terraform/ndelius-test-automation.tf:22]: I just mean if I add someone's MOJ email, but they don't use GitHub, can they give their authorisation via some other channel?

Comment (NickWalt01) [reply terraform/ndelius-test-automation.tf:22]: ah sorry, yes, email or slack is fine

Comment (marcus-bcl) [reply terraform/ndelius-test-automation.tf]: Great, I've updated the `added_by` field.  Nic is having issues with Slack today, so if you do need to contact her for approval would it be possible to do that via email?  Thanks.

**Review by AntonyBishop — APPROVED** [review]

**Pull request merged**
